File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/Eksportforbud.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.kode.EksportUnntakCti;

import java.io.Serializable;

public class Eksportforbud implements Serializable {



    /**
     * Angir om personen har eksportforbud eller ikke.
     */
    boolean forbud;

    /**
     * Angir type eksportunntak.
     */
    EksportUnntakCti unntakType;


}

File: src/main/java/no/nav/pensjon/regler/domain/grunnlag/Eksportforbud.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.kode.EksportUnntakCti
import java.io.Serializable

class Eksportforbud : Serializable {
    /**
     * Angir om personen har eksportforbud eller ikke.
     */
    var forbud = false

    /**
     * Angir type eksportunntak.
     */
    var unntakType: EksportUnntakCti? = null
}